src/routes/mods.rs

```diff
@@ -46,7 +46,7 @@ pub async fn mods_get(
             body_url: m.body_url,
             published: m.published,
             updated: m.updated,
-            status: match m.status.0 {
+            status: match m.status {
```

Comment:
This is why I recommended adding a more robust system for this.  This currently treats any `Processing` mods as `Rejected`, which is incorrect.